src/Spout/Writer/XLSX/Helper/BorderHelper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Box\Spout\Writer\XLSX\Helper;
+
+use Box\Spout\Writer\Style\BorderPart;
+use Box\Spout\Writer\Style\Border;
+
+class BorderHelper
+{
+    public static $xlsxStyleMap = [
+        Border::STYLE_SOLID.'%'.Border::WIDTH_THIN => 'thin',
```

Comment:
instead of using a separator, can you use a multi-level array?

```php
public static $xlsxStyleMap = [
        Border::STYLE_SOLID => [
             Border::WIDTH_THIN => 'thin',
             Border::WIDTH_MEDIUM => 'medium',
             ...
        ],
        ...
];
```